scripts/react_native_pods.rb

```diff
@@ -62,6 +62,7 @@ def use_react_native! (options={})
   end
 
   if hermes_enabled
+    pod 'React-Core/Hermes', :path => "#{prefix}/"
```

Comment:
Do you need this to make "HelloWorld" to build with Hermes?
I thought the below `pod 'React-hermes'` is enough but I don't know pod enough to tell.

Reply:
see latest comment